templates/api-blueprint.swig

```diff
@@ -79,6 +79,10 @@ FORMAT: 1A
 
 {{ transition.description }}
 
+{% for copy in transition|getCopy %}
```

Comment:
Hmm.. You are right. But the code contains https://github.com/apiaryio/snowcrash/blob/master/src/Blueprint.h#L256 which is why I got confused in the first place.